topics/bazis.qmd

```diff
@@ -461,18 +510,88 @@ Combining this in a slurm script we can queue matlab workloads.
 
 * [mathworks-parpool](https://nl.mathworks.com/help/parallel-computing/parpool.html)
 
-## Connect to VU Yoda/iRODS with icommands
+## Acknowledging the cluster
+
+When you use the cluster for your research, please acknowledge the cluster in your publications. You can use the following text:
+
+   > We kindly acknowledge sponsoring by the VU HPC Council and the IT for Research (ITvO) BAZIS Linux computational cluster at the VU University Amsterdam.
```

Comment:
```suggestion
   > We kindly acknowledge sponsoring by the VU HPC Council and the IT for Research (ITvO) BAZIS Linux computational cluster at VU Amsterdam.
```

Otherwise it reads Vrije University University Amsterdam 🤔